org.hl7.fhir.utilities/src/main/resources/source/rendering-phrases-de.po

```diff
@@ -1,5580 +1,5593 @@
-msgstr ""
-
+msgid "Operation"
+msgstr ""
+
 # CAPABILITY_OTH_RES_ENB
 #: CAPABILITY_OTH_RES_ENB
-msgid "The other resources enabled for"
-msgstr ""
-
+msgid "The other resources enabled for"
+msgstr ""
+
 # CAPABILITY_PATCH_INT
 #: CAPABILITY_PATCH_INT
-msgid "PATCH a new resource version (patch interaction)"
-msgstr ""
-
+msgid "PATCH a new resource version (patch interaction)"
+msgstr ""
+
 # CAPABILITY_PROF_CONF
 #: CAPABILITY_PROF_CONF
-msgid "Profile Conformance"
-msgstr ""
-
+msgid "Profile Conformance"
+msgstr ""
+
 # CAPABILITY_PROF_MAP
 #: CAPABILITY_PROF_MAP
-msgid "Profile Mapping"
-msgstr ""
-
+msgid "Profile Mapping"
+msgstr ""
+
 # CAPABILITY_PROF_RES_DOC
 #: CAPABILITY_PROF_RES_DOC
-msgid "Document Resource Profile"
-msgstr ""
-
+msgid "Document Resource Profile"
+msgstr ""
+
 # CAPABILITY_PUB_BY
 #: CAPABILITY_PUB_BY
-msgid "Published by: {0}"
-msgstr ""
-
+msgid "Published by: {0}"
+msgstr ""
+
 # CAPABILITY_PUB_ON
 #: CAPABILITY_PUB_ON
-msgid "Published on: {0}"
-msgstr ""
-
+msgid "Published on: {0}"
+msgstr ""
+
 # CAPABILITY_READ_INT
 #: CAPABILITY_READ_INT
-msgid "GET a resource (read interaction)"
-msgstr ""
-
+msgid "GET a resource (read interaction)"
+msgstr ""
+
 # CAPABILITY_REF_PROF
 #: CAPABILITY_REF_PROF
-msgid "Reference Policy"
-msgstr ""
-
+msgid "Reference Policy"
+msgstr ""
+
 # CAPABILITY_REQ_RECOM
 #: CAPABILITY_REQ_RECOM
-msgid "Required and recommended search parameters"
-msgstr ""
-
+msgid "Required and recommended search parameters"
+msgstr ""
+
 # CAPABILITY_REST_CAPS
 #: CAPABILITY_REST_CAPS
-msgid "FHIR RESTful Capabilities"
-msgstr ""
-
+msgid "FHIR RESTful Capabilities"
+msgstr ""
+
 # CAPABILITY_REST_CONFIG
 #: CAPABILITY_REST_CONFIG
-msgid "REST Configuration: {0}"
-msgstr ""
-
+msgid "REST Configuration: {0}"
+msgstr ""
+
 # CAPABILITY_RES_CONF
 #: CAPABILITY_RES_CONF
-msgid "Resource Conformance: {0}"
-msgstr ""
-
+msgid "Resource Conformance: {0}"
+msgstr ""
+
 # CAPABILITY_RES_ENB
 #: CAPABILITY_RES_ENB
-msgid "The linked resources enabled for"
-msgstr ""
-
+msgid "The linked resources enabled for"
+msgstr ""
+
 # CAPABILITY_RES_OPER
 #: CAPABILITY_RES_OPER
-msgid "The operations on the resource (if any)"
-msgstr ""
-
+msgid "The operations on the resource (if any)"
+msgstr ""
+
 # CAPABILITY_RES_PRO
 #: CAPABILITY_RES_PRO
-msgid "Capabilities by Resource/Profile"
-msgstr ""
-
+msgid "Capabilities by Resource/Profile"
+msgstr ""
+
 # CAPABILITY_RES_TYP
 #: CAPABILITY_RES_TYP
-msgid "Resource Type"
-msgstr ""
-
+msgid "Resource Type"
+msgstr ""
+
 # CAPABILITY_REV_PROF
 #: CAPABILITY_REV_PROF
-msgid "The relevant profiles (if any)"
-msgstr ""
-
+msgid "The relevant profiles (if any)"
+msgstr ""
+
 # CAPABILITY_SEARCHES
 #: CAPABILITY_SEARCHES
-msgid "Searches"
-msgstr ""
-
+msgid "Searches"
+msgstr ""
+
 # CAPABILITY_SEARCH_INT
 #: CAPABILITY_SEARCH_INT
-msgid "GET all set of resources of the type (search interaction)"
-msgstr ""
-
+msgid "GET set of resources of the type (search interaction)"
```

Comment:
Why is CrowdIn editing existing translations? Is there some way to prevent this behaviour?